sql/core/src/main/scala/org/apache/spark/sql/execution/python/PythonArrowOutput.scala

```diff
@@ -46,6 +46,13 @@ private[python] trait PythonArrowOutput[OUT <: AnyRef] { self: BasePythonRunner[
 
   protected def arrowMaxRecordsPerOutputBatch: Int = SQLConf.get.arrowMaxRecordsPerOutputBatch
 
+  protected def arrowMaxBytesPerOutputBatch: Int = SQLConf.get.arrowMaxBytesPerOutputBatch.toInt
+
+  if (arrowMaxRecordsPerOutputBatch > 0 && arrowMaxBytesPerOutputBatch > 0) {
+    throw new IllegalArgumentException(
+      "Both max records and max bytes of an Arrow output batch cannot be set at the same time.")
```

Comment:
Is it possible to support both together in the future?

Reply:
It is possible if we need it.